### ISSUE TITLE
using Valve.VR isn't importing namespace

### ISSUE THREAD

**Issue by Glokta0** [ValveSoftware/steamvr_unity_plugin#292]
After importing the SteamVR-Plugin (assetstore or github), my Visual Studio is not able to work with the "using Valve.VR;". It doesn't throw an error, but it is not auto-completing SteamVR specific variables when I type. 

![image](https://user-images.githubusercontent.com/43697593/51560152-c2bf1e00-1e51-11e9-989a-75c422eb76b7.png)

Tried so far:
on multiple projects, 
multiple versions of SteamVR (2.0.1 from AssetStore and newest Beta)
opened the downloaded Beta version from the downloadlocation as a project directly.

Edit: OpenVR is installed

**Comment by zite** [comment]
Might try RC2 on the releases page, it has fixed a couple assembly definition things since the betas. If that doesn't work try and delete the .sln and .csproj files in the root of your unity project. Unity should rebuild them correctly.

**Comment by Glokta0** [comment]
Yeah, based on the long changelog, I used RC2 before. Just tested it with different versions of unity (2018.3.2f1, 2018.3.0f2, 2018.2.20f1) with RC2 - no fix. 
I inserted the Assets into assets and replaced project settings with those from the downloaded folder.

Unity rebuilt all the deleted files, but Visual Studio still doesn't understand anything from Valve.VR's namespace.

**Comment by Glokta0** [comment]
Welp. Problem found: wrong Unity settings, that presumably came with a reinstall after bigger unity update.
Just in case somebody stumbles as well:
![image](https://user-images.githubusercontent.com/43697593/51576692-3cbcca80-1e85-11e9-947f-d2c1bba4dfe1.png)
... was set to wrong editor.

Thanks so much!

**Comment by mlekoo** [comment]
> Welp. Problem found: wrong Unity settings, that presumably came with a reinstall after bigger unity update.
> Just in case somebody stumbles as well:
> ![image](https://user-images.githubusercontent.com/43697593/51576692-3cbcca80-1e85-11e9-947f-d2c1bba4dfe1.png)
> ... was set to wrong editor.
> 
> Thanks so much!

Thank you very much, you helped me a lot! ;)

**Comment by Giantbean** [comment]
This is happening to me with the proper IDE set using Unity 2019.3 with Open VR Desktop 2.0.5 and Steam VR2.2.0. I have regenerated project files for the .csproj from all sources. I have deleted .csproj and .sln as well as .vs files and allowed Unity to regenerate them. Saved closed rebooted etc but the Valve namespace and all sub namespaces fail giving me a repeating error:

" NullReferenceException: Object reference not set to an instance of an object
Valve.VR.InteractionSystem.Sample.LockToPoint.FixedUpdate () (at Assets/SteamVR/InteractionSystem/Samples/BuggyBuddy/LockToPoint.cs:54) "

The LockToPoint script is being called in another script which gives an error "The type or namespace 'LockToPoint' could not be found..." while the using Valve.VR.InteractionSystem.Sample; says "Using Directive is Unnecessary". Removing .Sample from the using removes the last error but the namespace remains grayed out and the reference to LockToPoint still through s errors?   

Things like // (SteamVR) Player reference
    public Valve.VR.InteractionSystem.Player player;
    public Valve.VR.InteractionSystem.Hand leftHand;
    public Valve.VR.InteractionSystem.Hand rightHand; 

Which have worked for months all fail now do to a missing assembly reference with using Valve.VR; present. 

Any idea whats going on or how to fix it?
